www/docs/server/metadata.md

```diff
@@ -53,15 +53,50 @@ const isAuthed = t.middleware(async ({ meta, next, ctx }) => {
 export const authedProcedure = t.procedure.use(isAuthed);
 
 export const appRouter = t.router({
-  hello: authedProcedure.meta({ hasAuth: false }).query(() => {
+  hello: authedProcedure.meta({ authRequired: false }).query(() => {
     return {
       greeting: 'hello world',
     };
   }),
-  protectedHello: authedProcedure.meta({ hasAuth: true }).query(() => {
+  protectedHello: authedProcedure.meta({ authRequired: true }).query(() => {
     return {
       greeting: 'hello-world',
     };
   }),
 });
 ```
+
+## Default Meta, Chaining, and shallow merging
+
+If you set Meta multiple times, it will be shallow merged.
```

Comment:
```suggestion
If you set meta multiple times, it will be shallowly merged.
```

Even though it's `interface Meta`, docs still use lower-cased version for them, eg. context for `type Context` - https://trpc.io/docs/context